tests/unit/editors/object.spec.js

```diff
@@ -76,4 +76,31 @@ describe('Object Editor', () => {
       expect(editor.getValue()).toEqual(spec.value)
     })
   })
+
+  it('with option maxRecursions 0, should create an editor (schema with recursion on object properties)', () => {

```

Comment:
Move this test in another file since here are tested specific fixtures. 
Also try adding a new test with a different recursion depth to test how many editors are created